runtime/src/identity.rs

```diff
@@ -166,7 +168,9 @@ pub struct SigningItemWithAuth {
 }
 
 /// The module's configuration trait.
-pub trait Trait: system::Trait + balances::Trait + timestamp::Trait {
+pub trait Trait:
+    system::Trait + balances::Trait + timestamp::Trait + group::Trait<group::Instance1>
```

Comment:
To access KYCServiceProviders group instance, instead of extending, a simpler way is to use a trait member. For e.g., in runtime:

```
impl staking::Trait for Runtime {
type KYCServiceProviders = KYCServiceProviders;
}
```
That way in staking we can define a member type:
```
pub trait Trait: system::Trait {
type KYCServiceProviders = GetMembers;
}
```
Now you can use `KYCServiceProviders::some_method()`. 

This is easier for testing too. I believe the intent behind making `Group::add_member` is to get this behavior.

Reply:
There are two things happening here one is extending and another is making method `add_member()` public. The intent is different for both

extending is used for identity module to access the list of members in a required function. while for testing, I have to make a dispatchable public so I can call it from another module test.

On your given solution 
How do I access the function without actually implementing the trait even I declared `KYCServiceProviders` in the runtime? 

& I don't understand how It helps me to access the private dispatchable?

Reply:
I think this is easier to implement when PR #166 is merged. Since that also introduces KYC providers in Identity. Let's leave it as is here.